sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -63,6 +151,21 @@ export interface OnMessage {
     (message: ServiceBusMessage): Promise<void>;
 }
 
+// @public
+export interface Queue extends QueueOptions {
```

Comment:
Suggestion: Rename this to something less generic. Otherwise, it can easily be confused with other types.

Reply:
Renamed to use `QueueDetails`, updated other names accordingly as well